azure_monitor/django_sample/readme.md

```diff
@@ -1,65 +1,30 @@
-
-## Utilize Function Apps with OpenCensus
-
-- Start the Azure Storage Emulator.
-- Switch to Visual Studio Code.
-- Open a terminal window, run the following commands to start the Python functions locally:
-
-```powershell
-cd .\docs_samples\Functions
-
-python -m pip install -r requirements.txt
-
-func start
-```
-
-- Browse to the HTTP function endpoint `http://localhost:7071/api/MyHttpFunction`.
-- Run the following command to deploy the function apps to Azure. Be sure to replace the `SUFFIX` below (you can find it in the deployed Azure Portal resource group or in the `.\docs_samples\suffix.txt` file that was generated by the setup script):
-
-```powershell
-cd .\docs_samples\Functions
-
-az login
-
-func azure functionapp publish python-appinsights-SUFFIX   
-```
-
-- You should see the two function applications deployed successfully:
-
-  ![The results of the function app publish command is displayed.](./media/python_functionapp_deploy.png "Review the results and ensure the function apps deployed succesfully.")
-
-- Switch to the Azure Portal.
-- Navigate to the **python-appinsights-SUFFIX** function app.
-- Under **Functions**, select **Functions**, you should see the two python function apps deployed.
-
-> **NOTE** Azure Function runtimes support various Python versions depending on the version selected.  See [Languages by runtime version](https://docs.microsoft.com/en-us/azure/azure-functions/supported-languages#languages-by-runtime-version).
-
-- Review the function app code.  Notice the usage of the a storage account and a call to the database.
-- Navigate to the **python-appinsights-SUFFIX** Application Insights resource, under **Monitoring**, select **Logs**
-- Run the following query:
-
-```kusto
-dependencies
-```
+This next example shows how to add the various OpenCensus settings to a Django application to send logging and metric data to Azure Monitor.  You will run the samples both locally and deploy the Django web application to the Azure App Service resource you deployed as part of the setup script.
 
-- You should see the dependencies displayed for the calls to MySQL and Azure storage displayed.
+## 1 - Environment Setup
 
-## Configuring Django applications
+Follow the steps in the [setup documentation](/azure_monitor/readme.md).
 
-This next example shows how to add the various OpenCensus settings to a Django application to send logging and metric data to Azure Monitor.  You will run the samples both locally and deploy the Django web application to the Azure App Service resource you deployed as part of the setup script.
+## 2 - Configure Environment
 
 - Run the following commands to setup the environment:
 
 ```powershell
 cd $home
-cd .\docs_samples\WebSample
+
+cd .\azure_monitor\django_sample
 ```
 
 ```python
```

Comment:
Any purpose to have the below installs explicitly if the `requirements.txt` already includes them? Also I believe the `opencensus.ext.azure` need to be `opencensus-ext-azure` with hyphens instead of periods in the `requirements.txt`.

Reply:
removed the extra steps.